shared/output/gapic/templates/garbage/lib/so/much/trash/garbage_service/client.rb

```diff
@@ -118,6 +118,54 @@ def initialize
 
           # Service calls
 
+          ##
+          # Retrieves an EmptyGarbage resource
+          #
+          # @overload get_empty_garbage(request, options = nil)
+          #   @param request [So::Much::Trash::EmptyGarbage | Hash]
+          #     Retrieves an EmptyGarbage resource
+          #   @param options [Gapic::CallOptions, Hash]
+          #     Overrides the default settings for this call, e.g, timeout, retries, etc. Optional.
+          #
+          # @overload get_empty_garbage()
```

Comment:
This usage doesn't work. (The `request` parameter is required.) We probably need to get rid of this overload.

One might argue that, for an empty request, we should do away with the `request` parameter altogether. Unfortunately, that doesn't work because we'd be stuck if optional fields were added to the request proto in the future. It's possible that we could set the default for `request = {}`. I'll think about the ramifications of that, but in the meantime, please just remove this overload.